### DISCUSSION TITLE
[Office 365] Domyślny język

### DISCUSSION THREAD

**Discussion by cabana19** [AzureCommunityPL/group#351] [Pomoc] [answered]
Hej
Mam pytanko, nowa organizacja w Office na nowy tenant, nowo tworzone konta w M365 mają domyślnie język polski, chciałbym żeby miały angielski. Jak to zrobić? 😉 
Dziękuję!

**Comment by sbugalski** [comment, marked as answer]
Ten artykuł powinien rozwiać Twoje wątpliwości: https://blog.thinformatics.com/2020/09/microsoft-365-language-confusion/
